imagemanager/imagemanager.go

```diff
@@ -81,13 +81,19 @@ type CryptoContext interface {
 	CreateSignContext() (fcrypt.SignContextInterface, error)
 }
 
+// StateStorage provides API to remove state storage.
```

Comment:
Comment doesn't correspond to interface. Name it `StateRemover`, or `StateStorageRemove`. Good article how to name interfaces in go: https://medium.com/@dotronglong/interface-naming-convention-in-golang-f53d9f471593

Reply:
Done